docs/users/community-operators-troubleshooting.md

```diff
@@ -0,0 +1,229 @@
+
+### <a id="resolve-pr-type"></a>resolve-pr-type
+Failures at this stage are rare. To diagnose the issue, review the pipeline logs linked in the PipelineRun summary within the PR. If the logs don’t clarify the problem, feel free to ask for assistance in the PR comments.  Maintainers will assist in identifying and resolving the issue.
+
+### <a id="apply-test-waivers"></a>apply-test-waivers
+Failures at this stage are rare. To diagnose the issue, review the pipeline logs linked in the PipelineRun summary within the PR. If the logs don’t clarify the problem, feel free to ask for assistance in the PR comments.  Maintainers will assist in identifying and resolving the issue.
+
+### <a id="content-hash"></a>content-hash
+Failures at this stage are rare. To diagnose the issue, review the pipeline logs linked in the PipelineRun summary within the PR. If the logs don’t clarify the problem, feel free to ask for assistance in the PR comments.  Maintainers will assist in identifying and resolving the issue.
+
+### <a id="certification-project-check"></a>certification-project-check
+Failures at this stage are rare. To diagnose the issue, review the pipeline logs linked in the PipelineRun summary within the PR. If the logs don’t clarify the problem, feel free to ask for assistance in the PR comments.  Maintainers will assist in identifying and resolving the issue.
+
+### <a id="get-organization"></a>get-organization
+Failures at this stage are rare. To diagnose the issue, review the pipeline logs linked in the PipelineRun summary within the PR. If the logs don’t clarify the problem, feel free to ask for assistance in the PR comments.  Maintainers will assist in identifying and resolving the issue.
+
+### <a id="get-pyxis-certification-data"></a>get-pyxis-certification-data
+Failures at this stage are rare. To diagnose the issue, review the pipeline logs linked in the PipelineRun summary within the PR. If the logs don’t clarify the problem, feel free to ask for assistance in the PR comments.  Maintainers will assist in identifying and resolving the issue.
+
+### <a id="static-tests"></a>static-tests
+Failures at this stage are rare. To diagnose the issue, review the pipeline logs linked in the PipelineRun summary within the PR. If the logs don’t clarify the problem, feel free to ask for assistance in the PR comments.  Maintainers will assist in identifying and resolving the issue.
+
+### <a id="static-tests-results"></a>static-tests-results
+If the static tests fail, a summary will be posted as a comment on the Pull Request, providing detailed reasons for the failure. The summary of test results will look like as shown below:
+
+![Preflight test run logs example](../img/static_tests_example.png)
+
+To proceed:
+1.	Review the comment for the detailed reasons behind the failed static tests.
+1.	Fix all the reported issues.
+1.	Re-trigger the hosted pipeline using the command: `/pipeline restart operator-hosted-pipeline`
```

Comment:
If they fix the issue and re-push the commit it will automatically restarts a pipeline.